src/platform/silabs/rs911x/wfx_sl_ble_init.cpp

```diff
@@ -294,6 +294,7 @@ void SilabsBleWrapper::rsi_ble_add_char_val_att(void * serv_handler, uint16_t ha
 
 uint32_t SilabsBleWrapper::rsi_ble_add_matter_service(void)
 {
+    bleEvent.eventData                                      = new SilabsBleWrapper::sl_wfx_msg_t();

```

Comment:
Instead of using dynamic allocation, we could just change the structure type to avoid using a pointer for the eventData. From i can tell, there doesn't seem to be an explicitly requirement and the variable is global.